ingestion-beam/src/main/java/com/mozilla/telemetry/util/Normalize.java

```diff
@@ -124,4 +125,27 @@ public static String appName(String name) {
     return OTHER;
   }
 
+  /**
+   * Normalize a 2-letter country code value to uppercase, otherwise return "Other".
+   *
+   * <p>This function is intended to be used to normalize ISO 3166-1 alpha-2 codes, but we do not
+   * check that the code actually corresponds to an officially assigned country so that we don't
+   * need to worry about our list of codes drifting out of date. The space of two-letter codes
+   * (26 * 26 = 676) is sufficiently small that the impact of invalid codes would likely be small.
```

Comment:
Note this difference from the existing implementation.

Reply:
This seems reasonable to me.